content/en/docs/kitex/Tutorials/framework-exten/registry.md

```diff
@@ -73,4 +76,14 @@ Specify your own registration module and customized registration information thr
   svr := xxxservice.NewServer(handler, server.WithRegistry(yourRegistry), server.WithRegistryInfo(yourRegistryInfo))
   ```
 
+- Allow custom registry to get the actual address for registering
```

Comment:
`to get the address specified by `WithRegistryInfo`, instead of the listening address.`